README.md

```diff
@@ -162,6 +162,8 @@ you to manage and maintain those images and containers in a production environme
 familiar container cli commands.  For more details, see the
 [Container Tools Guide](https://github.com/containers/buildah/tree/master/docs/containertools).
 
+![Podman Hello](https://podman.io/images/podman-hello.jpg)
```

Comment:
Can we just copy the plain text here instead of a picture.
The text is to small on mobile and white text on a blue background is not well readable.

Reply:
+1 for the plain text. Also; the image doesn't show what command generated that response.

Reply:
The first line is the command